lib/google/gax/api_callable.rb

```diff
@@ -222,8 +222,8 @@ def page_token
     # @raise [StandardError] if +settings+ has incompatible values,
     #   e.g, if bundling and page_streaming are both configured
     def create_api_call(func, settings, params_extractor: nil)
-      api_caller = proc do |api_call, request|
-        api_call.call(request)
+      api_caller = proc do |api_call, request, _settings, block|
+        api_call.call(request, block)
       end
 
       if settings.page_descriptor
```

Comment:
Passing a block for metadata should probably raise a runtime exception for paginated calls.

Reply:
Ping on this.